src/en/charms-upgrading.md

```diff
@@ -0,0 +1,28 @@
+Title: Upgrading applications
+
+# Upgrading applications
+
+An application is upgraded by upgrading its corresponding charm. Since a charm
+is always associated with a *channel* and a charm's version can fluctuate
+within that channel the following rules apply:
+
+- If the channel has a version **newer** than the deployed version then the latter
+  is **upgraded** to the former.
+- If the channel has a version **older** than the deployed version then the latter
+  is **downgraded** to the former.
+
```

Comment:
It might be useful to give some example here

Reply:
Pain.